docs/en-US/component/time-picker.md

```diff
@@ -43,48 +43,52 @@ time-picker/range
+| readonly              | whether TimePicker is read only                          | ^[boolean]                                                                                      | false       |
+| disabled              | whether TimePicker is disabled                           | ^[boolean]                                                                                      | false       |
+| editable              | whether the input is editable                            | ^[boolean]                                                                                      | true        |
+| clearable             | whether to show clear button                             | ^[boolean]                                                                                      | true        |
+| size                  | size of Input                                            | ^[enum]`'large' \| 'default' \| 'small'`                                                        | —           |
+| placeholder           | placeholder in non-range mode                            | ^[string]                                                                                       | —           |
+| start-placeholder     | placeholder for the start time in range mode             | ^[string]                                                                                       | —           |
+| end-placeholder       | placeholder for the end time in range mode               | ^[string]                                                                                       | —           |
+| is-range              | whether to pick a time range                             | ^[boolean]                                                                                      | false       |
+| arrow-control         | whether to pick time using arrow buttons                 | ^[boolean]                                                                                      | false       |
+| popper-class          | custom class name for TimePicker's dropdown              | ^[string]                                                                                       | —           |
+| range-separator       | range separator                                          | ^[string]                                                                                       | '-'         |
+| format                | format of the displayed value in the input box           | ^[string] see [date formats](/en-US/component/date-picker#date-formats)                         | —           |
+| default-value         | optional, default date of the calendar                   | ^[Date] / ^[array]`[Date, Date]`                                                                | —           |
+| id                    | same as `id` in native input                             | ^[string] / ^[array]`[string, string]`                                                          | -           |
```

Comment:
<img width="1004" alt="image" src="https://user-images.githubusercontent.com/27342882/220606204-cf2a25e0-0201-41b5-9011-f96a1f6a046c.png">

the default value of id needs to be changed! (`-` > `—`)

Reply:
You're right.